examples/lightrag_oracle_demo.py

```diff
@@ -97,6 +97,7 @@ async def main():
             graph_storage="OracleGraphStorage",
             kv_storage="OracleKVStorage",
             vector_storage="OracleVectorDBStorage",
+            addon_params={"example_number": 1, "language": "Simplfied Chinese"},
```

Comment:
In the demo, let's keep the default language as English. You can modify the README to add usage instructions.

Reply:
I have fixed follow your advice.
Information about all parameters is added into the readme.